src/main/java/com/prgms/vouchermanager/validation/InputValidation.java

```diff
@@ -0,0 +1,38 @@
+package com.prgms.vouchermanager.validation;
+
+public class InputValidation {
+
+
+    public boolean validFrontMenu(int menu) {
+        if (menu == 1 || menu == 2||menu==3) {
+            return true;
+        }return false;
+    }
+
+
+    public boolean validVoucherMenu(int menu) {
+        if (menu == 1 || menu == 2) {
+            return true;
+        }return false;
+    }
+
+    public boolean validCustomerMenu(int menu) {
+        if (menu == 1) {
+            return true;
+        }
+        return false;
+    }
+
+    public boolean validVoucherType(int number) {
+        if (number==1||number==2) {
+            return true;
+        }else return false;
+    }
+
+    public boolean validPercent(long discountValue) {
+        if (discountValue > 100) {
+            return false;
+        }
+        return  true;
+    }
```

Comment:
```suggestion
    public boolean validFrontMenu(int menu) {
        return menu == 1 || menu == 2 || menu == 3;
    }

    public boolean validVoucherMenu(int menu) {
        return menu == 1 || menu == 2;
    }

    public boolean validCustomerMenu(int menu) {
        return menu == 1;
    }

    public boolean validVoucherType(int number) {
        return number == 1 || number == 2;
    }

    public boolean validPercent(long discountValue) {
        return discountValue <= 100;
    }
```

조건문으로 true / false 를 반환하지 않아도 되 보입니다.